### ISSUE TITLE
Air compression (I think that's what I'm talking about)

### ISSUE THREAD

**Issue by Chris353536345373344** [studio-minus/ppg-bugs-and-requests#1458]
A new mechanic that when objects are going fast enough they start to "compress" air thus starting to increase in tempature, this increase scales with speed

It would allow "space" themed things to be more advanced needing heat shields (or just cooling elements) to cool when falling, Although this might need a higher gravity to allow enough speed to start heating. This should also be an option to toggle

Additional context 
Basically if going over let's say 1000 meters per second it gains a couple of degrees (farenheight) per second

**Comment by username90001** [comment]
aerodynamic wont be added because of idk it wont be added but there are mods that give every object aerodynamics
